fabfile/add.py

```diff
@@ -22,7 +27,14 @@
                                        "you make a request, basically reloading the code. Very han",
                                        "dy when developing. Set to 0 for unlimited requests.: "))),
     ("ssl_enabled", "Is SSL enabled on this server (yes/no)?: "),
-    ("git_branch", "From which git branch will the server pull the project?: ")
+    ("git_branch", "From which git branch will the server pull the project?: "),
+    ("vault_used", "Are you utilizing a vault to store secrets for this server (yes/no)?: ")
```

Comment:
it's asking whether you want to use one or not. If you say yes, then you choose which vault file (which has the credentials in json format) you want to go with. If no, then yurt just fills in the secrets fields with hardcoded secrets. I would want to make sure that if there are no vaults available, to not show this option.